src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/Example4.java

```diff
@@ -0,0 +1,30 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidEscapedUnicodeCharacters">
+      <property name="allowIfAllCharactersEscaped" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+// xdoc section -- start
+public class Example4 {
+  // OK, a normal String below
+  String unitAbbrev = "μs";
+  // violation below, not all characters are escaped ('s'). 'should be avoided.'
+  String unitAbbrev1 = "\u03bcs";
+  String unitAbbrev2 = "\u03bc\u03bc\u03bc";
```

Comment:
Add comment above, as sometimes there is violation

Reply:
Comments should be above each line, in all examples.
Location of them is not changing, content of them is changing.

Reply:
`// sometimes there is violation` right ?

Reply:
`// ok, because ....`